src/index.js

```diff
@@ -84,6 +84,14 @@ var Accordion = React.createClass({
     setTimeout(this._getContentHeight);
   },
 
+  componentWillReceiveProps(nextProps) {
+    // when content prop changes, update content height
+    this.setState({
```

Comment:
Do we need to set `content` on state here?  Looks to me like simply calling `_getContentHeight()` would get the job done.